tests/CppWinRTAuthoringTests.cpp

```diff
@@ -206,57 +206,70 @@ TEST_CASE("CppWinRTAuthoringTests::EventsAndCppWinRt", "[property]")
 #endif // WINRT_Windows_Foundation_H
 
 #if defined(WINRT_Windows_UI_Xaml_Data_H)
+#include <winrt/Windows.System.h>
 #include <winrt/Windows.UI.Xaml.Hosting.h>
 
-// This test cannot run in the same process as the malloc spies tests in wiTest.cpp
-// MSFT_internal: https://task.ms/44191550
 TEST_CASE("CppWinRTAuthoringTests::NotifyPropertyChanged", "[property][LocalOnly]")

```

Comment:
```suggestion
TEST_CASE("CppWinRTAuthoringTests::NotifyPropertyChanged", "[property]")
```

The `[LocalOnly]` tag disables for CI. Presumably this can be enabled now